src/esp/gfx/PbrShader.h

```diff
@@ -131,18 +131,10 @@ class PbrShader : public Magnum::GL::AbstractShaderProgram {
      */
     NormalTexture = 1 << 6,
 
-    /**
```

Comment:
Just curious, why was this removed? (My guess: your new version of the shader will always use a scale, so you didn't need this flag anymore.)

Reply:
Yep. Talked to mosra about it and it's sufficiently cheap to always enable.